frontend/web/src/app/stl-viewer/classes/LoaderSelector.ts

```diff
@@ -0,0 +1,39 @@
+/**
+	@class LoaderSelector
+ */
+export class LoaderSelector {
+    loaderTable = {}
+
+    /**
+        Register a scene loader for a specific file format, using the file extesion name for lookup.
+        @param {String} fileExtName extension name for the specific file format.
+        @param {Function} loaderCtor constructor of the loader class.
+        */
+    registerLoader(fileExtName, loaderCtor) {
+        if((typeof loaderCtor) == 'function') {
+            this.loaderTable[fileExtName] = loaderCtor;
+        }
+    }
+
+    /**
+        Get the proper loader for a target file format using the file extension name.
+        @param {String} fileExtName file extension name for the specific format.
+        @returns {Object} loader object for the specific format; null if not found.
+        */
+    getLoader(fileExtName) {
+        var loaderCtor = this.loaderTable[fileExtName.toLowerCase()];
+        if(!loaderCtor)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') if statements must be braced [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS121)